lib/commands/project/project.js

```diff
@@ -363,3 +385,110 @@ function spawn(command, options, cwd, cb) {
     cb(err);
   });
 }
+
+
+//.option('-p, --path-name [path]', 'a sepecific path of api')
+//.option('-f, --testmodule <module>', 'one of: ' + testmodules)
+//.option('-t, --assertion-format <type>', 'one of: ' + assertiontypes)
+//.option('-s, --asynchronous <boolen>', 'one of: ' + istrue)
+//.option('-o, --separate <boolean>', 'one of: ' + istrue)
+function testGenerate(options, cb) {
+  var template = require('swagger-test-templates');
+  var desiredPaths = [];
+  var testmodule = 'supertest';
+  var assertionFormat = 'expect';
+  var asynchronous = true;
+  var separate = false;
+
+  // parse swagger.yaml, get the json object
+  var yaml = require('js-yaml');
+  var string = fs.readFileSync('./api/swagger/swagger.yaml', {encoding: 'utf8'});
```

Comment:
Don't use a relative path like this - it will vary based on where you run the command. In fact, don't assume you know where the file is at all! Take a look at this for an example of how you should find the swagger file: https://github.com/swagger-api/swagger-node/blob/master/lib/commands/project/project.js#L202.